src/docker-compose.dcproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>fea0c318-ffed-4d39-8781-265718ca43dd</ProjectGuid>
     <DockerLaunchBrowser>True</DockerLaunchBrowser>
-    <DockerServiceUrl>http://localhost:5100</DockerServiceUrl>
+    <DockerServiceUrl>http://docker.for.win.localhost:5100</DockerServiceUrl>
```

Comment:
The address should be `host.docker.internal`, as docker.for.win.localhost is now deprecated.
```suggestion
    <DockerServiceUrl>http://host.docker.internal:5100</DockerServiceUrl>
```